config/software/chef-provisioning-fog.rb

```diff
@@ -24,11 +24,13 @@
   dependency "ruby-windows-devkit"
 else
   dependency "ruby"
-  dependency "rubygems"
 end
 
-dependency "bundler"
+dependency "rubygems"
 dependency "nokogiri"
+dependency "bundler"
+dependency "chef"
```

Comment:
I made every plugin depend on chef to ensure that we never install a plugin or its dependencies before we install all of chef.